Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUExternalTextureDescriptor.h

```diff
@@ -35,22 +36,14 @@ typedef struct __CVBuffer* CVPixelBufferRef;
 
 namespace WebCore::WebGPU {
 
-struct HTMLVideoElementIdentifier {
-    uint64_t identifier;
-};
-struct WebCodecsVideoFrameIdentifier {
-    std::pair<uint64_t, uint64_t> identifier;
-};
-
-using VideoSourceIdentifier = std::variant<HTMLVideoElementIdentifier, WebCodecsVideoFrameIdentifier>;
-
 struct ExternalTextureDescriptor : public ObjectDescriptorBase {
-    VideoSourceIdentifier mediaIdentifier;
+    uint64_t mediaIdentifier;
```

Comment:
Now that what was PAL is moved to WebCore, we should be able to use WebCore::MediaIdentifier here and at related places